tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_operational_pii_fields_test.py

```diff
@@ -0,0 +1,103 @@
+"""
+Integration test for RemoveOperationalPiiFields
+"""
```

Comment:
It would be nice if you included the issue number.

```suggestion
"""
Integration test for RemoveOperationalPiiFields

Original Issue: DC-1220
"""
```